src/Backend/Modules/Blog/Actions/DeleteCategory.php

```diff
@@ -51,12 +51,12 @@ public function execute(): void
                 BackendModel::createURLForAction('Categories') . '&report=deleted-category&var=' .
                 rawurlencode($this->record['title'])
             );
```

Comment:
still missing a return here, below a this->redirect if it is not the last line of the method should be a return, in the old code that wasn't present becaus it used the if else structure thereby making it the "last" line of executed code